rosidl_parser/rosidl_parser/parser.py

```diff
@@ -234,6 +234,7 @@ def extract_content_from_ast(tree):
                 namespaces=goal_request.structure.type.namespaces,
                 name=goal_request_basename),
             goal_request, result_response, feedback_message)
+        content.elements += action.implicit_includes
```

Comment:
This might result in a duplicate include though.

Reply:
Why not add the deduplication here (rather than in potentially every location which uses this API)?